### PR TITLE
Bugfix enhanced pagespeed insights

### PR DESCRIPTION
**Pull request by Jakub89**
Resolves: #81

### CODE REVIEW

File: company_website/templates/main_page.haml [hunk 1]
```diff
@@ -29,9 +29,12 @@
...
-    %script{:async => "",
-        :defer => "",
-        :src => "https://maps.googleapis.com/maps/api/js?key={{ google_api_key }}&callback=initMap"}
+    :javascript
+      if(screen.width >= 720) {
+          document.write(
+              '<script src="https://maps.googleapis.com/maps/api/js?key={{ google_api_key }}&callback=initMap" async defer><'+
+              '/script>'
+          );
+      }
```
Comment (kbeker) [inline company_website/templates/main_page.haml]: This should be in `.js` file. Please move it there

File: company_website/static/main_page/map_navigator.sass [hunk 2]
```diff
@@ -1,3 +1,7 @@
+@import "../common/partials/constants"
 .map
```
Comment (kbeker) [inline company_website/static/main_page/map_navigator.sass]: Please add empty line after import :)

File: company_website/templates/main_page.haml (hunk 1 above)
Comment (Jakub89) [reply company_website/templates/main_page.haml]: It can't because of `{{ google_api_key }}` variable :/

Comment (kbeker) [reply company_website/templates/main_page.haml]: Try to add `js` variable in haml file and then move `js` code to separate file

Comment (Jakub89) [reply company_website/templates/main_page.haml]: Done

File: company_website/static/main_page/map_navigator.sass (hunk 2 above)
Comment (Jakub89) [reply company_website/static/main_page/map_navigator.sass]: Done

File: company_website/templates/main_page.haml [hunk 3]
```diff
@@ -27,11 +27,9 @@
 {% endblock %}
 
 {% block extra_script %}
+    :javascript
+      let google_api_key = "{{google_api_key}}";
```
Comment (kbeker) [inline company_website/templates/main_page.haml]: @Jakub89 just FYI, please add always space between variable and `{{ ... }}` :)

**Pull request merged**